src/projects/update-project.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class UpdateProjectDTO {
+    title: string;
+    type: string;
+    description: string;
+    observations: [''];
```

Comment:
Not sure about the quotation marks here - it could affect nothing or it could add an observation as a null string. Have you seen any issues?

Ideally this would be typed as an Observation[ ]

Reply:
Just had it engrained from the form declarations in the front end.  Pulled them out.